README.md

```diff
@@ -0,0 +1,319 @@
+4. Set `PANGEA_KONG_CONFIG_FILE` to Pangea Kong Config file location. This file should be mounted in the `volumes` section of the `compose.yaml` file. In above example it is set to `/etc/pangea_kong_config.json` due to in `volumes` sections we have:
+    ```yaml
+        volumes:
+            - ./config/pangea_kong_config.json:/etc/pangea_kong_config.json
+    ```
+    This means that `./config/pangea_kong_config.json` in our local machine is mounted into `/etc/pangea_kong_config.json` on the Pangea Plugin docker. The Pangea Kong Plugin configuration file will be documented in a later section.
+
+
+#### Volumes
+
+To allow docker to access a local file we should declare it on `volumes` sections into the `compose.yaml` file.
+```yaml
+    volumes:
+      - <origin>:<destination>
+```
+It is possible to map this origin from whatever location in our local machine to whatever destination into the docker but it is important to take care about location permissions. Also the `destination` path should be saved into the environment variable `PANGEA_KONG_CONFIG_FILE` so the plugin could load it.
+In above example relative path `./config` make reference to the root directory where `compose.yaml` file is.
+
+
+#### Secrets
+
+```yaml
+secrets:
+  pangea_ai_guard_token_secret:
+    file: ./secrets/pangea_ai_guard_token.txt
+```
+
+This means that the secret called `pangea_ai_guard_token_secret` will point to the local file `./secrets/pangea_ai_guard_token.txt`. In this case the relative path `./secrets` make reference to the root directory where `compose.yaml` file is. Secrets are mounted into the container as a file in `/run/secrets/<secret_name>`. For more information [click here](https://docs.docker.com/compose/how-tos/use-secrets/).
+
+
+### Pangea Plugin Config File
+
+```json
+{
+  "pangea_domain": string,      # Pangea Domain got from Pangea Console
+  "insecure": string,           # set to false to use http conections
+  "rules": [
+    {
+      "host": string,           # host that send the request to Kong AI gateway
+      "endpoint": string,       # Kong AI gateway endpoint used to send the request
+      "prefix": string,         # Kong AI gateway endpoint prefix. It will be removed from the endpoint
+      "protocols": array,       # List of string with protocols allowed
+      "ports": array,           # List of string with ports allowed 
+      "allow_on_error": bool,   # Whether or not request will reach the LLM in case AI Guard request fails
+      "parser": string,         # Parser name used to translate the LLM request to Pangea AI guard format. i.e.: 'openai'
+      "ai_guard": {
+        "request": {            # Pangea AI Guard default parameters
+          "parameters": {
+            "recipe": string    # Pangea AI Guard recipe
```

Comment:
`#` for comments does not display nicely
![image](https://github.com/user-attachments/assets/956625b0-2a8b-4588-b6d4-70157f718b90)

Reply:
Actually it might be the `string` and `bool` parts.

Reply:
Yes, because as json it's expecting a quoted string. But it's for the purpose of documentation. Do you have another idea? We can change it.

Reply:
We can write sample values, maybe even the defaults, for each setting. For example, instead of writing `bool`, write `false`. Then it's legal. The strings can be empty strings or `"string"` if there is no default.

As for the comments, maybe `//` will work? If not, I wonder if `jsonc` instead of `json` is available as a language tag.

Reply:
Much better now. Maybe we try `jsonc` next?

![image](https://github.com/user-attachments/assets/7a3973ed-7f02-4df1-a1c1-ba8673a2281b)

Reply:
Hadn't pushed `jsonc` tag because didn't see any change on VS Code but it seems to work on GitHub web, doesn't it?